Lacework/lacework_module/base.py

```diff
@@ -0,0 +1,17 @@
+from pydantic import BaseModel, Field
+from sekoia_automation.connector import Connector
+from sekoia_automation.module import Module
+
+
+class LaceworkConfiguration(BaseModel):
+    secret_key: str = Field(..., description="Your unique identifier used as the Authorization header")
```

Comment:
As `secret_key` is a secret, you have to set `secret=True` in order to ask the engine to decrypt this property.
```suggestion
    secret_key: str = Field(secret=True, description="Your unique identifier used as the Authorization header")
```